src/util/db/sqlite.cpp

```diff
@@ -0,0 +1,48 @@
+#include "util/db/sqlite.h"
+
+#include "util/assert.h"
+
+namespace {
+
+/// Date/time formate generated by SQLite CURRENT_TIMESTAMP (UTC) that is
+/// used for library.last_played_at and PlalistTracks.pl_datetime_added.
+/// QStringBuilder cannot be used to compose this constant at compile time!
+const QString kGeneratedTimestampFormat =
+        QStringLiteral("yyyy-MM-dd hh:mm:ss"); // date + separator + time
+const QString kGeneratedTimestampDateFormat = QStringLiteral("yyyy-MM-dd");
+const QString kGeneratedTimestampTimeFormat = QStringLiteral("hh:mm:ss");
+const QChar kGeneratedTimestampDateTimeSeparator = QChar(' ');
+
+} // namespace
+
+namespace sqlite {
```

Comment:
we should probably make this mixxx::sqlite so there isn't confusion that this is an official sqlite namespace